languages.yaml

```diff
@@ -2501,6 +2501,27 @@ Xml:
       - white
       - green
     chip: '#0060AC'
+XSL:
+  type: markup
+  ascii: |
+    {0}
+    {0}                     :::
+    {0}          ::::       :::  ::::
+    {0}        ::::::      :::   ::::::
+    {0}       :::::        :::     :::::
+    {0}     ::::::        :::       ::::::
+    {0}   ::::::          :::         ::::::
+    {0}   ::::::         :::          ::::::
+    {0}     ::::::       :::        ::::::
+    {0}       :::::     :::        :::::
+    {0}        ::::::   :::      ::::::
+    {0}          ::::  :::       ::::
+    {0}                :::
+    {0}
```

Comment:
```suggestion
    {0}                       :::
    {0}       :::::          :::  :::::
    {0}     :::::           :::     :::::
    {0}   :::::            :::        :::::
    {0} :::::             :::           :::::
    {0}:::::             :::             :::::
    {0} :::::           :::             :::::
    {0}   :::::        :::            :::::
    {0}     :::::     :::           :::::
    {0}       :::::  :::          :::::
    {0}             :::
```

Cleaned it a bit:

![image](https://user-images.githubusercontent.com/13710835/193618434-4caf4fdf-7393-4d6f-8fae-8de753a50a62.png)